inference-engine/ie_bridges/c/samples/hello_classification/README.md

```diff
@@ -1,31 +1,103 @@
+>
+> * Before running the sample with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](../../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
 >
-> The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
+> * The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
 
 You can do inference of an image using a trained AlexNet network on a GPU using the following command:
 
 ```sh
-./hello_classification_c <path_to_model>/alexnet_fp32.xml <path_to_image>/cat.bmp GPU
+./hello_classification_c <path_to_model>/alexnet_fp32.xml <path_to_image>/cat.png GPU
 ```
 
 ## Sample Output
 
 The application outputs top-10 inference results.
+
+```
+Top 10 results:
+
+Image /opt/intel/openvino/deployment_tools/demo/car.png
+
+classid probability
+------- -----------
+479     0.7562205
+511     0.0760381
+436     0.0724111
+817     0.0462140
+656     0.0301231
+661     0.0056171
+581     0.0031622
+468     0.0029917
+717     0.0023081
+627     0.0016193
+
+This sample is an API example, for any performance measurements please use the dedicated benchmark_app tool
+```
+
+## See Also
+* [Integrate the Inference Engine with Your Application](../../../../../docs/IE_DG/Integrate_with_customer_application_new_API.md)
+* [Using Inference Engine Samples](../../../../../docs/IE_DG/Samples_Overview.md)
+* [Model Downloader](@ref omz_tools_downloader_README)
+* [Model Optimizer](../../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md)
+
+[ie_core_create]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Core.html#gaab73c7ee3704c742eaac457636259541
+[ie_core_read_network]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Core.html#gaa40803295255b3926a3d1b8924f26c29
+[ie_network_get_input_name]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Network.html#ga36b0c28dfab6db2bfcc2941fd57fbf6d
+[ie_network_set_input_precision]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Network.html#gadd99b7cc98b3c33daa2095b8a29f66d7
+[ie_network_get_output_name]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Network.html#ga1feabc49576db24d9821a150b2b50a6c
+[ie_network_get_output_precision]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Network.html#gaeaa7f1fb8f56956fc492cd9207235984
+[ie_core_load_network]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Core.html#ga318d4b0214b8a3fd33f9e44170befcc5
+[ie_exec_network_create_infer_request]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__ExecutableNetwork.html#gae72247391c1429a18c367594a4b7db9f
+[ie_blob_make_memory_from_preallocated]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__Blob.html#ga7a874d46375e10fa1a7e8e3d7e1c9c9c
+[ie_infer_request_set_blob]:https://docs.openvinotoolkit.org/latest/ie_c_api/group__InferRequest.html#ga891c2d475501bba761148a0c3faca196
```

Comment:
I suppose such hardcoded links will be obsolete after new generation of docs, is not it?

CC @ntyukaev

Reply:
DO NOT MERGE until @ntyukaev provide a decision for C API links.

Reply:
We decided to merge changes as is and update links after changes in doc (PR @ntyukaev).